README.md

```diff
@@ -149,7 +149,7 @@ jobs:
       - run: rebar3 ct
 ```
 
-### Example (Gleam + `rebar3`, on Ubuntu)
```

Comment:
Do we want to get rid of the Gleam + `rebar3` example? I don't mind that much, but I don't understand why.

Reply:
Gleam switched to using its own binary as a build tool as the default, this likely will be the best supported way of building Gleam projects. And I thought it'd be weird having three examples specific to Gleam

Reply:
Sure thing. That makes sense.